Flesh out 'Deploy', sidequest to update 'Get Started' (#540)
elastic/docs-content: get-started/index.md

  - https://www.elastic.co/guide/en/elasticsearch/reference/current/index.html
  - https://www.elastic.co/guide/en/starting-with-the-elasticsearch-platform-and-its-solutions/current/index.html
---
# Get started

{{es}} is the heart of the [Elastic Stack](the-stack.md). Combined with {{kib}}, it powers these Elastic solutions and use cases:

* [Observability](/solutions/observability.md)
* [Security](/solutions/security.md)
* [Search](/solutions/search.md)

:::{tip}
Refer to our [customer success stories](https://www.elastic.co/customers/success-stories) for concrete examples of how Elastic is used in real-world scenarios.
:::

> ## First steps

Reviewer: Consider something like "Next steps?"  "First steps" sounds like the user has a long way to go.  "Next steps" implies that the user is already making progress.

> Depending on your experience with Elastic, you might want to start with one of these resources:

Reviewer: ```suggestion
Get started by learning about:
```

> - Learn about [{{es}} and {{kib}}](introduction.md), the core components of the Elastic Stack.
>   - Learn about [the stack](/get-started/the-stack.md) to understand the relationship between core and optional components of an Elastic deployment.
> - Learn about the [out-of-the-box solutions and use cases](/solutions/index.md) that Elastic supports.
> - Learn how to [deploy Elastic](./deployment-options.md) for your use case.
> - Understand how [versioning and availability](./versioning-availability.md) work with Elastic deployments.

Reviewer: ```suggestion
- [{{es}} and {{kib}}](introduction.md), the core components of the {{{stack}}}.
  - [the stack](/get-started/the-stack.md) to understand the relationship between core and optional components of an Elastic deployment.
- [out-of-the-box solutions and use cases](/solutions/index.md) that Elastic supports.
- [deploy Elastic](./deployment-options.md) for your use case.
- [versioning and availability](./versioning-availability.md) in Elastic deployments.
```
